runtime/oti/vmconstantpool.xml

```diff
@@ -146,6 +146,8 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 
 	<classref name="jdk/internal/foreign/abi/ProgrammableInvoker" flags="opt_openjdkFfi" versions="16-"/>
 
+	<classref name="java/lang/invoke/MethodHandleNatives" flags="opt_openjdkMethodhandle" versions="18-"/>
```

Comment:
Since no code is added to use this, is it really needed here?

Reply:
This `< classref/>` is required to generate `J9VMCONSTANTPOOL_JAVALANGINVOKEMETHODHANDLENATIVES`.

Reply:
Ah right. :)